### PR TITLE
Build 8.1.2

### PR DESCRIPTION
**Pull request by peastman**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by peastman** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
@peastman, the lints should be fixed before merging.

**Comment by peastman** [comment]
This is a patch release, which means nothing gets changed beyond what is absolutely necessary.  Since those are just suggestions, not errors, they'll wait for the next major release.

**Comment by peastman** [comment]
I merged the PR, but it hasn't triggered new builds.

**Comment by peastman** [comment]
@jaimergp I'm trying to follow your instructions to force it to trigger by pushing an empty commit, but they no longer work.  I can create an empty commit in my local repository, but pushing it with `git push upstream main` no longer creates a commit in the upstream repo.  How can I get it to trigger?

**Comment by peastman** [comment]
One build failed.  It looks like it was probably a network error.  Can someone rerun that one?  I don't have permission to do it.

**Comment by mattwthompson** [comment]
Is this meant to be a full release or another RC? I'm seeing that all builds are tagged with `conda-forge/label/openmm_rc`

image


https://anaconda.org/conda-forge/openmm/files?&version=8.1.2

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -2,6 +2,6 @@ opencl_impl:  # [osx]
 - apple       # [osx]
 - khronos     # [osx]
 channel_targets:
-- conda-forge main
+- conda-forge openmm_rc
```
Comment (jaimergp) [inline recipe/conda_build_config.yaml:5]: Yea it looks like this sneaked in. Gotta revert and rerender in a new PR.

Comment (mattwthompson) [reply recipe/conda_build_config.yaml:5]: I'll take a stab at this

Comment (peastman) [reply recipe/conda_build_config.yaml:5]: Oops!  Thanks for catching that!  And thanks Matt, that would be great.

**Pull request merged**